TRIST/docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
         ipv4_address: 192.168.1.10
     privileged: true
     volumes:
-      - ../:/TRIST
```

Comment:
Why have the volumes for the containers been changed? The directory above (`../`) contains all the required scripts for the containers to run properly... Unless I'm missing something?

Reply:
Please use your original version. i think it is accidentally changed.

Reply:
the pictures and models i've added are located at my anonmalydetection branch, would you be able to merge all of them into your main branch?